plugin/octo.vim

```diff
@@ -57,6 +57,7 @@ au BufEnter octo://* call s:configure_octo_buffer()
 au BufReadCmd octo://* lua require'octo'.load_buffer()
 au BufWriteCmd octo://* lua require'octo'.save_buffer()
 au CursorHold octo://* lua require'octo.reviews'.show_comment()
+au CursorHold octo://* lua require'octo'.show_summary()
```

Comment:
Nitpick: I think function names should express what you do. In this case it should first check if there is something to preview and then show it. So I would suggest to extend the name with something like `attempt_to_show_summary` or `eventually_show_summary` or `show_summary_if_cursor_on_issue`. I guess you get the point. Evaluate yourself if you like this idea. :wink:

Reply:
Im ok to change the name. Will think of a better name